BlockEntity/BEFruitpress.cs

```diff
@@ -519,9 +525,12 @@ private void udpateSqueezeRel(RunningAnimation anim)
             float selfHeight = (float)(juiceableLitresTransfered + juiceableLitresLeft) / 10f;
 
             squeezeRel += Math.Max(0, 0.9f - selfHeight);
+            pressSqueezeRel = GameMath.Clamp(squeezeRel, 0.1f, 1f);
             squeezeRel = GameMath.Clamp(Math.Min(mashStack.Attributes.GetDouble("squeezeRel", 1), squeezeRel), 0.1f, 1f);
 
             mashStack.Attributes.SetDouble("squeezeRel", squeezeRel);
+
+            screwPercent = GameMath.Clamp(1f - anim.CurrentFrame / (anim.Animation.QuantityFrames - 1), 0, 1f) / selfHeight;

```

Comment:
A quick calculation of how much the mash is squeezed. Dividing by selfHeight makes it so smaller mash responds more like expected while still always reaching 0% at the end.